### PR TITLE
3.0-ify devguide/templates (data binding helpers)

### PR DESCRIPTION
**Pull request by katejeffreys**
https://p3-data-binding-helper-dot-polymer-project.appspot.com/3.0/docs/devguide/templates

### CODE REVIEW

File: app/3.0/docs/devguide/templates.md [hunk 1]
```diff
@@ -13,10 +13,49 @@ data binding use cases:
...
 
-**2.0 tip.** The data binding helper elements are bundled in to the backward-compatible,
-`polymer.html` import. If you aren't using the legacy import, you'll need to import the
-helper elements you're using.
-{.alert .alert-info}
+The data binding helper elements are not included in the main Polymer library, so you need to import them before using them. For example:
+
+```js
+//import dom-repeat helper element
+import '@polymer/polymer/lib/elements/dom-repeat';
```
Comment (katejeffreys) [inline app/3.0/docs/devguide/templates.md]: Needs .js

File: app/3.0/docs/devguide/templates.md [hunk 2]
```diff
@@ -13,10 +13,49 @@ data binding use cases:
...
+...
+static get template(){
+  return html`
+    ...
+    <dom-bind ...> ... </dom-bind>
+    ...
+  `;
+}
+...
+```
```
Comment (katejeffreys) [inline app/3.0/docs/devguide/templates.md]: Syntax should be <template is="dom-thing"></template>

File: app/3.0/docs/devguide/templates.md [hunk 3]
```diff
@@ -57,53 +56,42 @@ library.
...
-
-    <div> Employee list: </div>
-    <template is="dom-repeat" items="{{employees}}">
-        <div># [[index]]</div>
-        <div>First name: <span>[[item.first]]</span></div>
-        <div>Last name: <span>[[item.last]]</span></div>
-    </template>
-
-  </template>
+[See it in Plunker](https://plnkr.co/edit/4HOkRP?p=preview)
```
[the remark this answers is not in the document]
Comment (arthurevans) [reply app/3.0/docs/devguide/templates.md]: Actuallly--read through and realized the colon/no colon isn't consistent. So let's just remove them.

File: app/3.0/docs/devguide/templates.md [hunk 4]
```diff
@@ -129,62 +117,56 @@ this.notifyPath('employees');
...
+import { PolymerElement, html} from '@polymer/polymer/polymer-element.js';
+import '@polymer/polymer/lib/elements/dom-repeat.js';
+
+class XCustom extends PolymerElement {
+  static get properties() {
+    return {
+      menuItems: {
+        type: Array,
+        value() {
+          return [
```
[the remark this answers is not in the document]
Comment (arthurevans) [reply app/3.0/docs/devguide/templates.md]: Kevin & Justin agreed we should move default values to constructor.  But OK to merge this PR like this; we can make a separate pass later if that's easier.

**Pull request merged**